From 7ce00bf77d1b92981f471a26fa2d92c96d14e51c Mon Sep 17 00:00:00 2001
From: Hans de Goede <hdegoede@redhat.com>
Date: Tue, 14 Jan 2025 23:07:35 +0100
Subject: [PATCH 5/9] media: ipu6: Fix out of tree builds

Fix out of tree builds not finding the ipu6*.h files.

Signed-off-by: Hans de Goede <hdegoede@redhat.com>
---
 drivers/media/pci/intel/ipu6/psys/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/media/pci/intel/ipu6/psys/Makefile b/drivers/media/pci/intel/ipu6/psys/Makefile
index fa638fcdf..9a3aa6e78 100644
--- a/drivers/media/pci/intel/ipu6/psys/Makefile
+++ b/drivers/media/pci/intel/ipu6/psys/Makefile
@@ -23,6 +23,8 @@ obj-$(CONFIG_VIDEO_INTEL_IPU6)		+= intel-ipu6-psys.o
 
 ifeq ($(is_kernel_lt_6_10), 1)
 ccflags-y += -I$(src)/../ipu6/
+else
+ccflags-y += -I/lib/modules/$(KERNELRELEASE)/build/drivers/media/pci/intel/ipu6/
 endif
 ccflags-y += -I$(src)/../
 ccflags-y += -I$(src)/../../
-- 
2.48.1

